src/stormd/service.rs

```diff
@@ -155,6 +154,9 @@ impl esb::Handler<ServiceBus> for Runtime {
             (ServiceBus::Rpc, BusMsg::Rpc(msg), ServiceId::Client(client_id)) => {
                 self.handle_rpc(endpoints, client_id, msg)
             }
+            (ServiceBus::Rpc, BusMsg::Storm(msg), other_source) => {
```

Comment:
The solution works, but I am not sure about the way.

I added this handle because the `bucketd` is marked as `ServiceId::Other`. Is this correct, or is it necessary to add the specific ServiceId to `bucketd`?